Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -5084,3 +5084,715 @@ nyseliffe,ZI,future,CBOT 5000 Oz Silver Futures,USD,5000,0.001,1
+bybit,AMBUSDT,cryptofuture,AMBUSDT,USDT,1,0.00001,10,AMBUSDT,10
+bybit,ANKRUSDT,cryptofuture,ANKRUSDT,USDT,1,0.00001,1,ANKRUSDT,1
+bybit,ANTUSDT,cryptofuture,ANTUSDT,USDT,1,0.005,0.1,ANTUSDT,0.1
+bybit,APEUSDT,cryptofuture,APEUSDT,USDT,1,0.001,0.1,APEUSDT,0.1
+bybit,API3USDT,cryptofuture,API3USDT,USDT,1,0.001,0.1,API3USDT,0.1
+bybit,APTUSDT,cryptofuture,APTUSDT,USDT,1,0.0005,0.01,APTUSDT,0.01
+bybit,ARBUSDT,cryptofuture,ARBUSDT,USDT,1,0.0001,0.1,ARBUSDT,0.1
+bybit,ARKMUSDT,cryptofuture,ARKMUSDT,USDT,1,0.00005,1,ARKMUSDT,1
+bybit,ARKUSDT,cryptofuture,ARKUSDT,USDT,1,0.00001,1,ARKUSDT,1
+bybit,ARPAUSDT,cryptofuture,ARPAUSDT,USDT,1,0.00001,10,ARPAUSDT,10
+bybit,ARUSDT,cryptofuture,ARUSDT,USDT,1,0.001,0.1,ARUSDT,0.1
+bybit,ASTRUSDT,cryptofuture,ASTRUSDT,USDT,1,0.000005,1,ASTRUSDT,1
+bybit,ATAUSDT,cryptofuture,ATAUSDT,USDT,1,0.00005,10,ATAUSDT,10
+bybit,ATOMUSDT,cryptofuture,ATOMUSDT,USDT,1,0.001,0.1,ATOMUSDT,0.1
+bybit,AUCTIONUSDT,cryptofuture,AUCTIONUSDT,USDT,1,0.001,0.1,AUCTIONUSDT,0.1
+bybit,AUDIOUSDT,cryptofuture,AUDIOUSDT,USDT,1,0.0001,0.1,AUDIOUSDT,0.1
+bybit,AVAXUSDT,cryptofuture,AVAXUSDT,USDT,1,0.005,0.1,AVAXUSDT,0.1
+bybit,AXSUSDT,cryptofuture,AXSUSDT,USDT,1,0.001,0.1,AXSUSDT,0.1
+bybit,BADGERUSDT,cryptofuture,BADGERUSDT,USDT,1,0.0001,0.1,BADGERUSDT,0.1
+bybit,BAKEUSDT,cryptofuture,BAKEUSDT,USDT,1,0.0005,0.1,BAKEUSDT,0.1
+bybit,BALUSDT,cryptofuture,BALUSDT,USDT,1,0.005,0.01,BALUSDT,0.01
+bybit,BANDUSDT,cryptofuture,BANDUSDT,USDT,1,0.001,0.1,BANDUSDT,0.1
+bybit,BATUSDT,cryptofuture,BATUSDT,USDT,1,0.0005,0.1,BATUSDT,0.1
+bybit,BCHUSDT,cryptofuture,BCHUSDT,USDT,1,0.05,0.01,BCHUSDT,0.01
+bybit,BELUSDT,cryptofuture,BELUSDT,USDT,1,0.0001,1,BELUSDT,1
+bybit,BICOUSDT,cryptofuture,BICOUSDT,USDT,1,0.0001,0.1,BICOUSDT,0.1
+bybit,BLURUSDT,cryptofuture,BLURUSDT,USDT,1,0.0001,1,BLURUSDT,1
+bybit,BLZUSDT,cryptofuture,BLZUSDT,USDT,1,0.00001,1,BLZUSDT,1
+bybit,BNBPERP,cryptofuture,BNBPERP,USD,1,0.05,0.01,BNBPERP,0.01
+bybit,BNBUSDT,cryptofuture,BNBUSDT,USDT,1,0.05,0.01,BNBUSDT,0.01
+bybit,BNTUSDT,cryptofuture,BNTUSDT,USDT,1,0.00005,1,BNTUSDT,1
+bybit,BNXUSDT,cryptofuture,BNXUSDT,USDT,1,0.0005,0.1,BNXUSDT,0.1
+bybit,BOBAUSDT,cryptofuture,BOBAUSDT,USDT,1,0.00005,0.1,BOBAUSDT,0.1
+bybit,BONKUSDT,cryptofuture,1000BONKUSDT,USDT,1,0.000001,100,1000BONKUSDT,100
+bybit,BSVUSDT,cryptofuture,BSVUSDT,USDT,1,0.05,0.01,BSVUSDT,0.01
+bybit,BSWUSDT,cryptofuture,BSWUSDT,USDT,1,0.0001,1,BSWUSDT,1
+bybit,BTC-06OCT23,cryptofuture,BTC-06OCT23,USD,1,0.50,0.001,BTC-06OCT23,0.001
+bybit,BTC-22SEP23,cryptofuture,BTC-22SEP23,USD,1,0.50,0.001,BTC-22SEP23,0.001
+bybit,BTC-24NOV23,cryptofuture,BTC-24NOV23,USD,1,0.50,0.001,BTC-24NOV23,0.001
+bybit,BTC-27OCT23,cryptofuture,BTC-27OCT23,USD,1,0.50,0.001,BTC-27OCT23,0.001
+bybit,BTC-29DEC23,cryptofuture,BTC-29DEC23,USD,1,0.50,0.001,BTC-29DEC23,0.001
+bybit,BTC-29MAR24,cryptofuture,BTC-29MAR24,USD,1,0.50,0.001,BTC-29MAR24,0.001
+bybit,BTC-29SEP23,cryptofuture,BTC-29SEP23,USD,1,0.50,0.001,BTC-29SEP23,0.001
```

Comment:
Sorry I hadn't realized these were here before, we should not add specific future contract expiration settings but just a generic canonical future ticker (same for the ETH below). It will be up to the future chain provider to fetch specific expirations, like we do for non crypto futures/options/etc, see for example CME futures entries here.

Reply:
To be safe mind rebasing from master? We just updated the binance SPDB  entries👍

Reply:
Rebased it and removed the entries. I removed the USDC contracts completely for now as they also need some different handling in the brokerage.